src/service/post.js

```diff
@@ -1,5 +1,6 @@
 import crashlytics from '@react-native-firebase/crashlytics';
 
+import anonymousApi from './anonymousConfig';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `anonymousApi` is a new addition. Ensure that the `anonymousConfig.js` file exists and exports an object that can be used to make API calls. Also, ensure that this configuration is correctly set up to handle requests to the anonymous endpoints.

```diff
+ import anonymousApi from './anonymousConfig';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->